data/egtea.py

```diff
@@ -15,11 +16,20 @@ class EGATEDataset(Dataset):
     """
 
     def __init__(self, dataset_path, transform=None, target_transform=None):
-        pass
 
+        self.dataset_path = dataset_path
```

Comment:
I don't see `dataset_path` used again in any of the methods, do we have to save it ?